### DISCUSSION TITLE
More copyright botherings

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#4376] [General]
@westnordost
Sorry for bothering you about this, but some of media that were likely created fully by you are missing copyright info. Can you confirm authorship info?

[parking_and_stopping_signs_overview.xcf](https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/street%20parking/parking_and_stopping_signs_overview.xcf) (map of parking traffic signs over world) - is it Natural Earth based? Or some simplified OSM data? Or something else?

![screen06](https://user-images.githubusercontent.com/899988/190173952-db8ad9fb-fa09-45cf-858e-527af5064430.png)
(posting here as it is a gimp project file)

---------------------------

And for this ones: are you sole author here? Or is eye/scissors/etc copied from some emoji set?

https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/note/add_photo_24dp.svg
https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/note/attach_gpx_24dp.svg

https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/undo/visibility.svg
https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/undo/split.svg
https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/ar/camera_measure_24dp.svg
https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/ar/hand_phone.svg

**Comment by westnordost** [comment]
I don't remember where that map is from, some mspaint-bucket-fill-friendly map I found on the net. Should be one of the first results. It's none of https://en.wikipedia.org/wiki/Wikipedia:Blank_maps#World_2 ,I wouldn't mind if you redid it with one of those and don't care enough to do it myself. (The picture is not used in the app, so it really quite doesn't matter IMO)

The rest is drawn by myself. The authors.txt explicitly states that it's done by me unless otherwise noted, I do really not understand why you put so much time into this.

**Reply by matkoniecz** [reply]
> the authors.txt explicitly states that it's done by me unless otherwise noted

the tricky part is that in multiple cases files were definitely made (and committed by others)

and in some cases like with this map it would be incorrect to assign 100% authorship to you (but maybe it is fine with all core graphics directly used in SC itself like https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/recycling/glass.svg )

> I do really not understand why you put so much time into this.

Maybe it is paranoid and not worth doing but I wanted to go through files and clarify their licensing status to be 100% kosher here

**Reply by westnordost** [reply]
Well currently, we are not at all kosher here, thanks to Google ARCore

**Reply by mnalis** [reply]
Regarding:

> The rest is drawn by myself. The authors.txt explicitly states that it's done by me unless otherwise noted, I do really not understand why you put so much time into this.

I too think it would be better to name authors explicitly for each image. Yes, it is some work initially, but it seems we have a volunteer, and the slight increase of that .txt file would be negligible compared to the rest of the project. And it is quite useful as it:
- removes ambiguity (i.e. was it _really_ done by Tobias, or was it simply overlooked)
- makes it easy/possible to automatically check if some media was added, but authors not provided
- the earlier it is fixed, the less work it is to check in the future (having been through trying to get a game into Debian with strict DFSG guidelines, having to go through hundreds of git commits semi-manually to get to the bottom of things was very time consuming). Also, people tend to forget such things with time, so unless it is documented early enough even the person who did the final version might sometimes not be able to tell if there was prior work or not.
